torch/fx/passes/_tensorify_python_scalars.py

```diff
@@ -207,6 +211,24 @@ def _sympy_interp(expr: sympy.Expr) -> MetaProxy:
                         node, tracer=tracer, fake_mode=fake_mode
                     )
 
+            # Specialize all dimensions that contain symfloats. Here's
+            # an example test that requires this:
+            # PYTORCH_OPINFO_SAMPLE_INPUT_INDEX=4 tlp python test/inductor/test_torchinductor_opinfo.py TestInductorOpInfoCUDA.test_comprehensive_nn_functional_interpolate_bicubic_cuda_float32 # noqa: B950
```

Comment:
s/tlp//

Reply:
This seems incredibly suspicious. You shouldn't ever have a SymFloat that comes out of tensor.shape....

Reply:
IIRC it was something like Floor(zf0 * 1.5) or something like that. I can run the repro again to get the exact condition if you'd like.

Reply:
Definitely a bug. FloorToInt should be used when it gets coerced to int, if that's the case.